src/aiida/cmdline/params/types/path.py

```diff
@@ -81,6 +82,8 @@ def convert(self, value, param, ctx):
         try:
             return super().convert(value, param, ctx)
         except click.exceptions.BadParameter:
+            if pathlib.Path(value).exists():
+                raise
```

Comment:
I don't think it's very nice to vomit the whole stacktrace to users terminal if they make a mistake. Is it possible to use click.echo.error or something like that here?

Reply:
This _is_ caught and just printed as:
```
Usage: verdi archive import [OPTIONS] [--] [ARCHIVES]...
Try 'verdi archive import --help' for help.

Error: Invalid value for '[ARCHIVES]...': Path 'test.txt' is not readable.
```